bundles/org.openhab.binding.oppo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -116,9 +125,13 @@
 
 	<channel-type id="time_display">
 		<item-type>Number:Time</item-type>
-		<label>Time Display (S)</label>
-		<description>The Playback Time Elapsed/Remaining in Seconds</description>
-		<state readOnly="true" pattern="%d %unit%"/>
+		<label>Time Display</label>
+		<description>The Playback Time Elapsed/Remaining</description>
```

Comment:
Can you also update the I18N properties after these changes?

Reply:
good catch, fixed now.